test/e2e/tests/transaction/change-assets.spec.js

```diff
@@ -0,0 +1,240 @@
+const {
+  defaultGanacheOptions,
+  withFixtures,
+  logInWithBalanceValidation,
+} = require('../../helpers');
+const FixtureBuilder = require('../../fixture-builder');
+const { SMART_CONTRACTS } = require('../../seeder/smart-contracts');
+
+describe('Change assets', function () {
+  if (!process.env.MULTICHAIN) {
```

Comment:
nit: if the testing framework used has a function to skip the test it could be interesting to use it so maybe it could show in a summary how many tests were skipped